### PR TITLE
add support for Codoon

### PR DESCRIPTION
**Pull request by ben-29**

### CODE REVIEW

File: scripts/codoon_sync.py [hunk 1]
```diff
@@ -144,16 +170,20 @@ def login_by_phone(self):
...
         r = self.session.post(
-            f"{self.base_url}/api/get_old_route_log",
+            # f"{self.base_url}/api/get_old_route_log",
+            f"{self.base_url}/api/get_new_route_log",
             data=payload,
             auth=self.auth.reload(payload),
         )
+        print(r.headers)
         print(r)
+        print(r.headers)
```
Comment (yihong0618) [inline scripts/codoon_sync.py]: drop the print

**Comment by yihong0618** [comment]
Cool
Please drop all the `print` if it is for testing, and the comments~

**Comment by ben-29** [comment]
removed and format codoon_sync.py with black

**Comment by yihong0618** [comment]
Thanks a lot, I will test now

**Pull request merged**